src/components/_proxied-dot-io/packer/remote-plugin-docs/server.ts

```diff
@@ -25,9 +25,14 @@ import rehypePrism from '@mapbox/rehype-prism'
 // which we need to shim cause of how we're fetching remote content here
 import shimRemoteIncludes from 'lib/shim-remote-includes'
 
-async function generateStaticPaths({ navDataFile, remotePluginsFile }) {
+async function generateStaticPaths({
+	navDataFile,
+	remotePluginsFile,
+	mainBranch = 'main',
```

Comment:
I've ensured `remote-plugin-docs` utils consistently accept `mainBranch` as an option.

This way, at the page file level, we can set distinct settings for `dot-io` and `dev-dot`.